File: NoteBase.Core/DTOs/NoteBooksDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NoteBase.Core.DTOs
{
    public class NoteBooksDto
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public int TopId { get; set; }
        public string Name { get; set; }
        public int OrderNumber { get; set; }
        public bool IsDeleted { get; set; }

    }
}

File: NoteBase.Android.Java/NoteBaseAndroidCore/src/main/java/com/veyselkaranitekgoz/notebaseandroid/core/DTOs/NoteBooksDto.java
package com.veyselkaranitekgoz.notebaseandroid.core.DTOs;

public class NoteBooksDto {
    private int Id;
    private String UserId;
    private int TopId;
    private String Name;
    private int OrderNumber;
    private bool IsDeleted;

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public String getUserId() {
        return UserId;
    }

    public void setUserId(String userId) {
        UserId = userId;
    }

    public int getTopId() {
        return TopId;
    }

    public void setTopId(int topId) {
        TopId = topId;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public int getOrderNumber() {
        return OrderNumber;
    }

    public void setOrderNumber(int orderNumber) {
        OrderNumber = orderNumber;
    }

    public bool getIsDeleted() {
        return IsDeleted;
    }

    public void setIsDeleted(bool isDeleted) {
        IsDeleted = isDeleted;
    }
}
